modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/portlet/action/ValidateRankingMVCResourceCommand.java

```diff
@@ -205,27 +219,39 @@ public ValidateRankingMVCResourceRequest(
 			ResourceRequest resourceRequest) {
 
 			_indexName = ParamUtil.getString(resourceRequest, "index-name");
+			_inactive = ParamUtil.getBoolean(resourceRequest, "inactive");
 			_resultsRankingUid = ParamUtil.getString(
 				resourceRequest, "resultsRankingUid");
 
 			_aliases = Arrays.asList(
 				ParamUtil.getStringValues(resourceRequest, PARAM_ALIASES));
+			_queryString = ParamUtil.getString(resourceRequest, PARAM_KEYWORDS);
```

Comment:
Let's remove the line break and alphabetize these.

Reply:
Let's also remove `PARAM_ALIASES` and `PARAM_KEYWORDS` and use the direct String since they aren't being used anywhere else. 

```java
_queryString = ParamUtil.getString(resourceRequest, "keywords");
```